mlflow/tensorflow.py

```diff
@@ -975,41 +985,52 @@ def _log_early_stop_callback_metrics(callback, history):
                     last_epoch = len(history.history[metric_key])
                     try_mlflow_log(mlflow.log_metrics, restored_metrics, step=last_epoch)
 
-    def fit(self, *args, **kwargs):
-        with _manage_active_run():
-            original = gorilla.get_original_attribute(tensorflow.keras.Model, "fit")
 
-            unlogged_params = ["self", "x", "y", "callbacks", "validation_data", "verbose"]
+    class FitPatch(PatchFunction):
```

Comment:
This is an incomplete integration with TensorFlow since the remaining functions use `wrap_patch` instead of the new `safe_patch`; it's intended to demonstrate the utility of `PatchFunction`'s `on_exception` class for performing cleanup operations (e.g., tensorboard directory removal) in the event of an unexpected exception in patch code.

cc @aarondav